lib/mito/io/externals.h

```diff
@@ -0,0 +1,27 @@
+// code guard
+#if !defined(mito_io_externals_h)
+#define mito_io_externals_h
+
+// externals
+#include <fstream>
+#include <unordered_map>
+
+#ifdef WITH_VTK
```

Comment:
I would put these vtk includes into a header file in `lib/mito/io/vtk/externals.h`. Then `externals.h` will be included in `lib/mito/io/vtk/public.h`, similarly to the headers structure above this subdirectory.

Reply:
Done